### DISCUSSION TITLE
Per 'git-diff' cli options for delta

### DISCUSSION THREAD

**Discussion by ldelossa** [dandavison/delta#1602] [Q&A] [answered]
Hello. 

My common case is to use delta with line numbers on. 
However, this means I can't copy and paste from git diff.

Is it possible to funnel arguments to delta via `git diff` in anyway?

**Comment by raisedadead** [comment, marked as answer]
Related discussion https://github.com/dandavison/delta/discussions/944

**Here is my solution:**

You can turn line-numbers off per invocation like so, if you are always having it on in your config.

```
git -c delta.line-numbers=false diff
```

**Reply by ldelossa** [reply]
thank you :)

**Comment by dandavison** [comment]
Hi @ldelossa and @raisedadead. In addition to what @raisedadead suggested, here's my personal current solution:

```zsh
delta-toggle() {
    eval "export DELTA_FEATURES=$(-delta-features-toggle $1 | tee /dev/stderr)"
}
```
where `-delta-features-toggle` is https://github.com/dandavison/tools/blob/main/python/-delta-features-toggle.


So

```
delta-toggle    # shows current features
delta-toggle s  # toggles side-by-side
delta-toggle l  # toggles line-numbers
```

**Reply by ldelossa** [reply]
Nice!
